src/Core/GeometryPrimitives/Point.h

```diff
@@ -54,6 +54,15 @@ class Point
   inline explicit Point(const Vector& v);
   inline Point(double x, double y, double z)
   { d_[0] = x; d_[1] = y; d_[2] = z; }
+
+  template <typename T>
+  inline Point(T x, T y, T z)
+  {
```

Comment:
```suggestion
  {
    BOOST_STATIC_ASSERT(std::is_arithmetic<T>::value);
```

Reply:
Add `#include <boost/static_assert.hpp>` around line 38 too

Reply:
@RubioJr9 Here's the change to merge in

Reply:
Sorry, I thought I already did this